feedback/static/html/feedback_instructor.html

```diff
@@ -9,7 +9,7 @@
             <th colspan="6" role="columnheader">
               <span class="d-flex align-items-center"><span>{% trans "Unit" %}</span></span>
             </th>
-            <th colspan="3" role="columnheader">
+            <th colspan="6" role="columnheader">
```

Comment:
I don't think increasing colspan here makes sense. It appears all colspan attributes here are out of sync and not the right way to set widths of columns. I think you should remove them and set width on each column using style tag e.g `style="width:30%"`

Reply:
I agree. I will remove the colspan here